reana_server/status.py

```diff
@@ -173,23 +173,15 @@ def users_directory_size(self):
 
     def shared_volume_health(self):
         """REANA shared volume health."""
-        cmd = ["df", "-h", SHARED_VOLUME_PATH]
+        cmd = ["df", "-h", "--output=used,size,pcent", SHARED_VOLUME_PATH]
         output = self.execute_cmd(cmd).splitlines()
-        headers = output[0].split()
-        values = output[1].split()
-        used_index = headers.index("Used")
-        available_index = headers.index("Avail")
-        use_percentage_index = headers.index("Use%")
-
-        return (
-            f"{values[used_index]}/{values[available_index]} "
-            f"({values[use_percentage_index]})"
-        )
+        used_size, total_size, used_percentage = output[1].split()
+
+        return f"{used_size}/{total_size} ({used_percentage})"
 
     def get_status(self):
         """Get status summary for REANA storage."""
         return {
-            "user_directory_size": self.users_directory_size(),
```

Comment:
Should I also delete the `users_directory_size()` function? As far as I can tell, it is not used anywhere after these changes.

Reply:
I think we should remove it together with `_get_path()`, since as you mentioned it's not being used